libfreerdp/core/nla.c

```diff
@@ -1453,20 +1462,23 @@ static int nla_decode_ts_request(rdpNla* nla, wStream* s)
 		return -1;
 	}
 
```

Comment:
From my PoV changes before are ok, but the next changes that save the result of decoding in `rc` are useless and make the more complex to read.

Reply:
didn´t like the handling of the return as boolean value (`!value`) so I used the `rc=fkt(); if (rc == 0) return;` style

Reply:
ok, changed to use similar style as before but check for `xxx() == 0` instead of using a return variable.